_data/publications.yml

```diff
@@ -240,13 +240,22 @@
   year: 2025
   preprint: https://doi.org/10.48550/arXiv.2410.08676
 
-- title: "<i> Prompt-with-Me: in-IDE Structured Prompt Management for LLM-Driven Software Engineering </i>"
+- title: "<i> Prompt-with-Me: in-IDE Structured Prompt Management for LLM-Driven Software Engineering</i>"
   authors:
     - Ziyou Li
     - Agnia Sergeyuk
     - Maliheh Izadi
   tracks:
     - 3
-  venue: 40th IEEE/ACM International Conference on Automated Software Engineering, (<strong>ASE 2025 -  Industry track</strong>)
+  venue: 40th IEEE/ACM International Conference on Automated Software Engineering, (<strong>ASE 2025 - Industry track</strong>)
   year: 2025
   preprint: https://doi.org/10.48550/arXiv.2509.17096
+
+- title: "<i>AgentGuard: Runtime Verification of AI Agents</i>"
+  authors:
+    - Roham Koohestani
+  tracks:
+    - 3
+  venue: Autonomous Agents in Software Engineering Workshop (<strong>AgenticSE Workshop</strong>), co-located with (<strong>ASE</strong>)
```

Comment:
You can add The First International Workshop on Autonomous Agents in Software Engineering Workshop